### PR TITLE
adding configmap custom integration doc

### PR DESCRIPTION
**Pull request by CharlyF**
### What does this PR do?
Example of how to enable an integration with a configmap.

### Motivation

More doc to better use the kubernetes integration.

### Additional Notes

Not sure if this should live here or in the `datadog-agent/Dockerfiles/agent/README.md` ?

### CODE REVIEW

File: content/integrations/kubernetes.md [hunk 1]
```diff
@@ -120,6 +120,48 @@ instances:
 
 See the [example kubernetes.yaml](https://github.com/DataDog/integrations-core/blob/master/kubernetes/conf.yaml.example) for all available configuration options.
 
+#### Custom integration
```
Comment (irabinovitch) [inline content/integrations/kubernetes.md]: Do we want to recommend this vs annotations?

Comment (CharlyF) [reply content/integrations/kubernetes.md]: Good point, I'd personally think we could document both.
From my experience however, configmaps are used to configure integrations and annotations are used for autodiscovery.

**Comment by l0k0ms** [comment]
@CharlyF could you merge this to https://github.com/DataDog/documentation/pull/2060 rather than master? many thanks 🙇  watch out, k8 set-up has its own section now.

**Comment by CharlyF** [comment]
Moved there https://github.com/DataDog/documentation/pull/2060/commits/8c33d64c95c06d65702939e28cb0ba8ce29cb3a3

**Pull request closed without merging**